src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Movie;
+
+public interface MovieDao {
+    Movie add(Movie movie) throws DataProcessingException;
```

Comment:
```suggestion
    Movie add(Movie movie);
```

this method should not throw any exceptions